<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cap_planning</class>
 <widget class="QWidget" name="cap_planning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_40">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_28" stretch="0,0,0,0">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>Data Collection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_41">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_54">
         <item>
          <widget class="QFrame" name="frame_94">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_95">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_52">
               <property name="topMargin">
                <number>9</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_22">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_43">
                  <item alignment="Qt::AlignTop">
                   <widget class="QLabel" name="label_19">
                    <property name="styleSheet">
                     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
                    </property>
                    <property name="text">
                     <string>Capacity Planning</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_2">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QFrame" name="frame">
                       <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                       </property>
                       <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                       </property>
                       <layout class="QVBoxLayout" name="verticalLayout_3">
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>0</number>
                        </property>
                        <item>
                         <widget class="QPushButton" name="pushButton_5">
                          <property name="text">
                           <string/>
                          </property>
                          <property name="icon">
                           <iconset resource="../../../resources.qrc">
                            <normaloff>:/dark_icon/images/dark_icon/help_24dp_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/dark_icon/images/dark_icon/help_24dp_FILL0_wght200_GRAD0_opsz24.png</iconset>
                          </property>
                          <property name="iconSize">
                           <size>
                            <width>24</width>
                            <height>24</height>
                           </size>
                          </property>
                          <property name="flat">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QPushButton" name="cap_table_loader">
                          <property name="styleSheet">
                           <string notr="true"/>
                          </property>
                          <property name="text">
                           <string/>
                          </property>
                          <property name="icon">
                           <iconset resource="../../../resources.qrc">
                            <normaloff>:/dark_icon/images/dark_icon/folder_open_24dp_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/dark_icon/images/dark_icon/folder_open_24dp_FILL0_wght200_GRAD0_opsz24.png</iconset>
                          </property>
                          <property name="iconSize">
                           <size>
                            <width>24</width>
                            <height>24</height>
                           </size>
                          </property>
                          <property name="flat">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QPushButton" name="cap_csv_save">
                          <property name="text">
                           <string/>
                          </property>
                          <property name="icon">
                           <iconset resource="../../../resources.qrc">
                            <normaloff>:/dark_icon/images/dark_icon/save_24dp_D4D4D4_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/dark_icon/images/dark_icon/save_24dp_D4D4D4_FILL0_wght200_GRAD0_opsz24.png</iconset>
                          </property>
                          <property name="iconSize">
                           <size>
                            <width>24</width>
                            <height>24</height>
                           </size>
                          </property>
                          <property name="flat">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                     <item>
                      <widget class="QFrame" name="frame_3">
                       <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                       </property>
                       <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                       </property>
                       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
                        <property name="spacing">
                         <number>0</number>
                        </property>
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>0</number>
                        </property>
                        <item>
                         <widget class="QFrame" name="frame_4">
                          <property name="styleSheet">
                           <string notr="true">background-color:transparent;</string>
                          </property>
                          <property name="frameShape">
                           <enum>QFrame::NoFrame</enum>
                          </property>
                          <property name="frameShadow">
                           <enum>QFrame::Raised</enum>
                          </property>
                          <layout class="QVBoxLayout" name="verticalLayout_5">
                           <property name="spacing">
                            <number>0</number>
                           </property>
                           <property name="leftMargin">
                            <number>0</number>
                           </property>
                           <property name="topMargin">
                            <number>0</number>
                           </property>
                           <property name="rightMargin">
                            <number>0</number>
                           </property>
                           <property name="bottomMargin">
                            <number>0</number>
                           </property>
                          </layout>
                         </widget>
                        </item>
                        <item>
                         <widget class="QFrame" name="frame_5">
                          <property name="frameShape">
                           <enum>QFrame::NoFrame</enum>
                          </property>
                          <property name="frameShadow">
                           <enum>QFrame::Raised</enum>
                          </property>
                          <layout class="QVBoxLayout" name="verticalLayout_6">
                           <property name="spacing">
                            <number>0</number>
                           </property>
                           <property name="leftMargin">
                            <number>0</number>
                           </property>
                           <property name="topMargin">
                            <number>0</number>
                           </property>
                           <property name="rightMargin">
                            <number>0</number>
                           </property>
                           <property name="bottomMargin">
                            <number>0</number>
                           </property>
                          </layout>
                         </widget>
                        </item>
                        <item>
                         <widget class="QFrame" name="frame_6">
                          <property name="frameShape">
                           <enum>QFrame::NoFrame</enum>
                          </property>
                          <property name="frameShadow">
                           <enum>QFrame::Raised</enum>
                          </property>
                          <layout class="QVBoxLayout" name="verticalLayout_7">
                           <property name="spacing">
                            <number>0</number>
                           </property>
                           <property name="leftMargin">
                            <number>0</number>
                           </property>
                           <property name="topMargin">
                            <number>0</number>
                           </property>
                           <property name="rightMargin">
                            <number>0</number>
                           </property>
                           <property name="bottomMargin">
                            <number>0</number>
                           </property>
                          </layout>
                         </widget>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_29">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_39">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="table_layout">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="maximumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="cap_info">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="styleSheet">
            <string notr="true">background-color:transparent;
border: none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot;&gt;Please upload planned generation capacities via .csv file. Csv files should follow the this format:&lt;/span&gt;&lt;/p&gt;
&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;
&lt;li style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Header Row: The first row of your CSV file should contain the following column headers: &lt;span style=&quot; font-family:'Courier New';&quot;&gt;Category&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;Type&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;Capacity Factor&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;2020&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;2021&lt;/span&gt;, &lt;span style=&quot; font-family:'Courier New';&quot;&gt;2022&lt;/span&gt;, etc.&lt;/li&gt;
&lt;li style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Data Rows&lt;/span&gt;: Each subsequent row should represent a data entry corresponding to the headers. For example, a row might look like this: &lt;span style=&quot; font-family:'Courier New';&quot;&gt;Phyd, Clean, 0.37, 1, 0, 0, etc.&lt;/span&gt;&lt;/li&gt;
&lt;li style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Category&lt;/span&gt;: The generation technology category, e.g., &lt;span style=&quot; font-family:'Courier New';&quot;&gt;Combustion Turbine&lt;/span&gt;.&lt;/li&gt;
&lt;li style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Type&lt;/span&gt;: Whether the generation is considered a clean or dirty source of energy, e.g., &lt;span style=&quot; font-family:'Courier New';&quot;&gt;Clean&lt;/span&gt;.&lt;/li&gt;
&lt;li style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Capacity Factor&lt;/span&gt;: The capacity factor of the energy source, e.g., &lt;span style=&quot; font-family:'Courier New';&quot;&gt;0.37&lt;/span&gt;.&lt;/li&gt;
&lt;li style=&quot; font-family:'Segoe UI'; font-size:9pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Yearly Data&lt;/span&gt;: The rated capacity in MW of each generation source for each year from &lt;span style=&quot; font-family:'Courier New';&quot;&gt;2020&lt;/span&gt; to &lt;span style=&quot; font-family:'Courier New';&quot;&gt;20&lt;/span&gt;xx e.g. &lt;span style=&quot; font-family:'Courier New';&quot;&gt;999, 888, etc.&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="hide_cap">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/dark_icon/images/dark_icon/remove_24dp_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/dark_icon/images/dark_icon/remove_24dp_FILL0_wght200_GRAD0_opsz24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_48">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="cc_cap_input_store">
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="next_7">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
